pkgs/top-level/all-packages.nix

```diff
@@ -41283,4 +41283,12 @@ with pkgs;
   waylyrics = callPackage ../applications/audio/waylyrics { };
 
   gitrs = callPackage ../tools/misc/gitrs { };
+
+  libvoikko = callPackage ../development/libraries/libvoikko { };
+
+  hfst = callPackage ../development/libraries/hfst { };
+
+  omorfi = callPackage ../applications/misc/omorfi { };
+
+  hfst-ospell = callPackage ../applications/misc/hfst-ospell { };
```

Comment:
Please sort these into fitting categories

Reply:
What would be the fitting categories? Should they all be under `development/libraries/`?